bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ChannelPool.java

```diff
@@ -53,20 +54,22 @@
 
   private static final AtomicInteger ChannelIdGenerator = new AtomicInteger();
 
+  public static ImmutableList<InstrumentedChannel>
```

Comment:
This seems like an internal implementation detail of of ChannelPool, might better to keep it private